src/pretix/static/pretixpresale/js/ui/main.js

```diff
@@ -82,7 +82,13 @@ var form_handlers = function (el) {
                     Math.abs(+new Date(opts.minDate) - new Date()) < Math.abs(+new Date(opts.maxDate) - new Date())
             ) ? opts.minDate : opts.maxDate;
         }
-        $(this).datetimepicker(opts);
+        $(this).datetimepicker(opts).on("dp.hide", function() {
```

Comment:
Can you add a small comment on why this code is necessary so we remember in a few years when we refactor this file?

Reply:
Done.